benchmark/sparse/linear_regression/mxnet_sparse_model.py

```diff
@@ -6,35 +6,50 @@
 import mxnet as mx
 
```

Comment:
add a reference to mxnet sparse linear regression example.
In case of future changes so it's easy to refer and maintain

Reply:
done